releases/evm/EVM-GMP-Release-Template.md

```diff
@@ -188,7 +188,13 @@ OPERATORS=$(cat "./axelar-chains-config/info/$ENV.json" | jq ".chains[\"$CHAIN\"
 ts-node evm/deploy-upgradable.js -c AxelarGasService -m [deployMethod] --args "{\"collector\": \"$OPERATORS\"}"
 ```
 
-9. Transfer ownership for Gateway, Operators and Gas Service contracts on `mainnet` and `testnet`
+8. Transfer ownership for Operators contract
+
+```bash
+ts-node evm/ownership.js -c Operators --action transferOwnership  --newOwner 0x9f5CDBc370B00C0dF52cf2619FA95907508108df
+```
+
+9. Transfer ownership for Gateway and Gas Service contracts on `mainnet` and `testnet`
```

Comment:
This is misleading, someone will transfer it to the stagenet operator before seeing step 9. Should be the same step but clearly mark the networks and corresponding address in a table